pyproject.toml

```diff
@@ -9,11 +9,16 @@ urls = { Repository = "https://github.com/robert-koch-institut/mex-model" }
 requires-python = "<3.13,>=3.11"
 dependencies = []
 optional-dependencies.dev = [
-    "black==24.8.0",
-    "ipdb==0.13.13",
-    "myst-parser==4.0.0",
-    "ruff==0.5.7",
-    "sphinx==7.4.7",
+    "black==24.8.0,<25",
+    "ipdb==0.13.1,<1",
+    "myst-parser==4.0.0,<5",
+    "mypy>=1.11.0,<2",
+    "pytest-cov>=5.0.0,<6",
+    "pytest-random-order>=1.1.1,<2",
+    "pytest-xdist>=3.6.1,<4",
+    "pytest>=8.3.1,<9",
```

Comment:
This repo has neither type checking nor tests.

```suggestion
```

Reply:
https://github.com/robert-koch-institut/mex-model/pull/107/commits/30678238ff52cda995e040f4dbecf8b2371e55fa